src/content/PostsContent.tsx

```diff
@@ -29,7 +29,7 @@ const posts: PostDescription[] = [
       'Visualizing processes in BPMN standard using the BPMN layout generator', // original title which would have been truncated: 'Visualizing processes in BPMN standard: using the BPMN layout generator to show activities, transitions, and gateways'
     text: 'In this article, we will use the BPMN layout generator to automatically generate a BPMN diagram from events logs. To further help visualize the process, we will add BPMN objects such as gateways.', // abstract in Bonitasoft Community: 'In this article, we will see how you can visualize event logs with the BPMN layout generator. We will use the BPMN layout generator to automatically generate a BPMN diagram from events logs. To further help visualize the process we will add BPMN objects such as gateways.'
     cover:
-      'https://lh5.googleusercontent.com/Kjqn5B3-PPYtJZ8k8iXtODqcIHS3AL3qlUWclH_ec-njXdCtZVnSFXz7ESRVILJCVLuXu047y_8tHxgaQ77miVPEqkoB3JIiLgpM1jqOjtQ1-8MLjGw2cfG_pFt5BaBSelY8r8GS',
+      'https://community.bonitasoft.com/sites/community.bonitasoft.com/files/blog/bonita-community/generateaprocessdiagramfromaneventlog.png',
```

Comment:
@csouchet I just realize that I choose the process map view where we are talking about BPMN layout in the description.
I don't remember the image we chose in the past, but I would prefer we use a BPMN view like in the following.
Do you agree? cc @oanesini 


![bpmn view](https://community.bonitasoft.com/sites/community.bonitasoft.com/files/blog/bonita-community/the_same_process_with_some_noise_filtered_out.png)

Reply:
Goog catch 😆 

Yes, it would be better to use BPMN view 🙂

Reply:
done in 13868da